### PR TITLE
Feature/initial environment setup

### PR DESCRIPTION
**Pull request by zollf**

### CODE REVIEW

**Review by joelfitzpat — DISMISSED** [review]
All good

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM python:3.8.3-alpine
```
Comment (Joel-Milligan) [inline Dockerfile]: Why is this for 3.8 rather than 3.9?

File: docker/django.dev.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FROM python:3.8.3
```
Comment (Joel-Milligan) [inline docker/django.dev.Dockerfile]: Why is this 3.8 vs 3.9?

File: setup.cfg [hunk 3]
```diff
@@ -0,0 +1,4 @@
+[pycodestyle]
+ignore = E302,E305
```
Comment (Joel-Milligan) [inline setup.cfg:2]: Why are we ignoring the 2 blank line requirement?

Comment (zollf) [reply setup.cfg:2]: coz it's gross

**Review by Joel-Milligan — COMMENTED** [review]
Why are the containers using 3.8 instead of 3.9?

File: setup.cfg (hunk 3 above)
Comment (Joel-Milligan) [reply setup.cfg:2]: fair enough

**Review by Joel-Milligan — APPROVED** [review]

**Pull request merged**